test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/acquisitiondetails/AcquisitionDetailsControllerSpec.scala

```diff
@@ -2515,4 +2517,51 @@ class AcquisitionDetailsControllerSpec
     )
   }
 
+  def validateAcquisitionDetailsCheckYourAnswersPage(
+    acquisitionDetailsAnswers: CompleteAcquisitionDetailsAnswers,
+    doc: Document
+  ): Unit = {
+    val expectedAcquisitionMethodDisplayName = acquisitionDetailsAnswers.acquisitionMethod match {
+      case AcquisitionMethod.Bought       => messages("returns.acquisitionMethod.Bought")
+      case AcquisitionMethod.Inherited    => messages("returns.acquisitionMethod.Inherited")
+      case AcquisitionMethod.Gifted       => messages("returns.acquisitionMethod.Gifted")
+      case AcquisitionMethod.Other(value) => value
+    }
+
+    doc.select("#acquisitionMethod-answer").text() shouldBe expectedAcquisitionMethodDisplayName
+    doc.select("#acquisitionPrice-answer").text() shouldBe formatAmountOfMoneyWithPoundSign(
+      acquisitionDetailsAnswers.acquisitionPrice.inPounds()
+    )
+
+    if (acquisitionDetailsAnswers.improvementCosts === AmountInPence.zero) {
```

Comment:
do we want to ensure that we're hitting all the `if` and `else` cases somehow by passing in certain data?